README.md

```diff
@@ -145,7 +145,6 @@ Give a ⭐️ if you like this project!
 
 ## 🔭 Future Features <a name="future-features"></a>
 
```

Comment:
- [ ] **Good job on using the Microverse README template 👍 👍** However, It would be better also to provide a valid link for your project's License in the README, we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish. 😎😎

Reply:
Fixed.